container/layer_tools.bzl

```diff
@@ -158,6 +158,8 @@ def incremental_load(
             ),
         ]
         if run:
+            # bazel automatically passes ctx.attr.args to the binary on run, so need to truncate
+            # them here to not pass twice, as they are already part of the docker entrypoint
```

Comment:
Can you add a pointer to the Bazel issue you opened so we can track and remove once that is fixed?

Reply:
This is also still a leftover from when I tried to also fix that issue. Yes will clean up.